security/keycloak-authz/src/main/java/io/quarkus/ts/openshift/security/keycloak/authz/AdminResource.java

```diff
@@ -20,13 +21,15 @@
 
     @GET
     @Produces(MediaType.TEXT_PLAIN)
+    @Authenticated
     public String get() {
         return "Hello, admin " + identity.getPrincipal().getName();
     }
 
     @GET
     @Path("/issuer")
     @Produces(MediaType.TEXT_PLAIN)
+    @Authenticated
```

Comment:
This annotation is no longer necessary. This was motivated by an issue that was fixed before 2.0.0.Final was out.